test/algconvergence/linear_method_tests.jl

```diff
@@ -58,6 +58,15 @@ test_setup = Dict(:alg=>Vern9(),:reltol=>1e-14,:abstol=>1e-14)
 sim = analyticless_test_convergence(dts,prob,CG2(),test_setup)
 @test sim.𝒪est[:l2] ≈ 2 atol=0.2
 
+A = DiffEqArrayOperator(ones(2,2),update_func=update_func)
```

Comment:
since this can in theory handle time and state, I think it should get a new test equation